lib/msf/base/serializer/readable_text.rb

```diff
@@ -5,13 +5,65 @@ module Serializer
 # This class formats information in a plain-text format that
 # is meant to be displayed on a console or some other non-GUI
 # medium.
-class ReadableText
+class ReadableText < Rex::Ui::Text::Output::Stdio
```

Comment:
To elaborate here, the Ui Output classes take care of what to do with the color codes so you don't have to worry about this at all.  Changing `ReadableText`'s superclass has no benefit and confuses the class' meaning.